servers/rendering/renderer_rd/shaders/forward_clustered/best_fit_normal.glsl

```diff
@@ -0,0 +1,43 @@
+#[compute]
```

Comment:
It might be worth adding a top-level comment to this shader to explain that this shader is only run once to generate an image on the GPU, rather than something run continuously.

Reply:
Done!